rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/40_range.yml

```diff
@@ -91,8 +91,84 @@ setup:
   - match: { aggregations.double_range.buckets.1.key: "0.0152-1.0" }
   - match: { aggregations.double_range.buckets.1.doc_count: 2 }
 
+---
+"Float range":
+  - skip:
+      version: " - 7.17.99"
```

Comment:
This should be `7.16.99` if you have merged #82435 first. Otherwise it should be `7.99.99` so it runs on `8.x`